lib/vsc/utils/run.py

```diff
@@ -396,7 +396,10 @@ def _wait_for_process(self):
 
     def _cleanup_process(self):
         """Cleanup any leftovers from the process"""
-        pass
+        try:
+            self._process.stdout.close()
```

Comment:
Do we need to check whether it's still open? What about `stderr`?

Reply:
I will add a check to see if it's still open and I haven't had any problems with `stderr` yet, but if you want I can do the same for `stderr`